nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-jetty/src/main/java/org/apache/nifi/web/server/JettyServer.java

```diff
@@ -715,20 +717,40 @@ private static int determineMaxWebRequestsPerSecond(NiFiProperties props) {
         return configuredMaxRequestsPerSecond > 0 ? configuredMaxRequestsPerSecond : defaultMaxRequestsPerSecond;
     }
 
+    private static long determineRequestTimeoutInMilliseconds(NiFiProperties props) {
```

Comment:
Recommend adding `final` modifier:
```suggestion
    private static long determineRequestTimeoutInMilliseconds(final NiFiProperties props) {
```